.github/workflows/docker-build-publish.yml

```diff
@@ -63,14 +68,15 @@ jobs:
 
       - name: Prepare
         id: prep
+        if: github.event.inputs.image-version == ''
```

Comment:
This `prepare` step will never executes when the `image-version` input is provided hence tags are not set. This means `tags: ${{ steps.prep.outputs.tags }}` will always be empty.

Reply:
My bad. I mad a mistake